docs/whatsnew/6.0.rst

```diff
@@ -18,6 +18,14 @@ PUT STATS HERE
 
 Removal of ``sunpy.database``
 -----------------------------
+
 The ``sunpy.database`` module has not been actively maintained for over a year now and has a number of outstanding issues.
 It has been deprecated since sunpy 4.1, and since there have been no major objections from the communityu since then we have completely removed ``sunpy.database`` in sunpy 6.0.
 If you are interested in seeing a replacement for ``sunpy.database``, either inside ``sunpy`` or as a third-party package, please join the discussion thread at https://community.openastronomy.org/t/deprecating-sunpy-database/495.
+
+Removal of ``sunpy.net.helioviewer.HelioViewerClient``
+------------------------------------------------------
+
+The Helioviewer Project maintains a Python wrapper around their API that supersedes the ``sunpy.net.helioviewer`` module.
+It is called `hvpy <https://hvpy.readthedocs.io/en/latest/>`__ and their documentation has several examples of how to use it.
+It has been deprecated since sunpy 4.1.
```

Comment:
```suggestion

``sunpy.net.helioviewer`` has been deprecated since sunpy v4.1 and has now been removed.
Users should instead use the `hvpy <https://hvpy.readthedocs.io/en/latest/>`__ package.
This package provides a Python wrapper around the Helioviewer API and is maintained by the Helioviewer Project.
The ``hvpy`` package supersedes the ``sunpy.net.helioviewer`` module.
```